src/config.rs

```diff
@@ -198,7 +199,9 @@ impl PgpPublicKey {
         let keyid = format_hex(cert.keyid().as_bytes(), "-", 4)?;
         let algo = cert.primary_key().pk_algo();
         let fpr = format_hex(cert.fingerprint().as_bytes(), " ", 2)?;
-        let p = policy::StandardPolicy::new();
+        let mut p = sequoia_policy();
+        p.accept_hash(sequoia_openpgp::types::HashAlgorithm::SHA1);
```

Comment:
`sequoia_policy` already calls `accept_hash(SHA1)`

Reply:
Good catch! That was a leftover from a previous attempt, will fix it soon.